### ISSUE TITLE
Web - Tooltip is not shown for a new user on invite message page

### ISSUE THREAD

**Issue by kbecciv** [Expensify/App#21946]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:
1. Go to staging dot on web chrome
2. Create a new workspace > Invite members > Click on invite > Select two members (For the First member, enter an email of a NEW user and the second user should be an old user)
3. Click on Next and hover over the profile avatars. Notice for the old user tooltip is shown, but if you hover over the avatar for the new user, tooltip is not shown.

## Expected Result:
Tooltip should be shown for a new user on invite message page

## Actual Result:
Tooltip is not shown for a new user on invite message page

## Workaround:
Unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android / native
- [ ] Android / Chrome
- [ ] iOS / native
- [ ] iOS / Safari
- [ ] MacOS / Chrome / Safari
- [ ] MacOS / Desktop

**Version Number:** 1.3.34-1
**Reproducible in staging?:** y
**Reproducible in production?:** y
**If this was caught during regression testing, add the test name, ID and link from TestRail:**
**Email or phone of affected tester (no customers):**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Notes/Photos/Videos:** Any additional supporting documentation

https://github.com/Expensify/App/assets/93399543/f23c6763-01d3-4256-9cb6-70793335cfd7

https://github.com/Expensify/App/assets/93399543/19ae808e-ece5-4b82-93c6-d9aeb2b9a033

**Expensify/Expensify Issue URL:**
**Issue reported by:** @priya-zha
**Slack conversation:** https://expensify.slack.com/archives/C049HHMV9SM/p1688021163521579

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by alitoshmatov** [comment]
## Proposal

### Please re-state the problem that we are trying to solve in this issue.
Web - Tooltip is not shown for a new user on invite message page
### What is the root cause of that problem?
`UserDetailsTooltip` is not working because only `accountID` was provided
https://github.com/Expensify/App/blob/619394df4bf5538cc0fc8d53c97db8bb2e5a18ea/src/components/MultipleAvatars.js#L186

### What changes do you think we should make in order to solve the problem?
We don't have a data for new users in `personalDetails` that's why tooltip cannot retrieve any data. But we can provide fallback data to display available information about user like this:
```javascript
fallbackUserDetails={{
                                    displayName: ReportUtils.getDisplayNameForParticipant(props.icons[1].id),
                                    login: lodashGet(props.icons[1], 'name',''),
                                    avatar: lodashGet(props.icons[1], 'source', ''),
                                }}
``` 
We should provide fallback data in every instance of tooltip in `MultipleAvatars`


### What alternative solutions did you explore? (Optional)

**Comment by NicMendonca** [comment]
still reviewing this

**Comment by NicMendonca** [comment]
@priya-zha are those clickable on other platforms? Should they be clickable? 🤔

**Comment by priya-zha** [comment]
No. It's not about being clickable or not. Just try to hover over the avatar and notice that on hover, the tooltip isn't present for new users.

**Comment by NicMendonca** [comment]
@priya-zha my apologies, I understand now. 

Any the other platforms? Can you make sure you are testing on all of those? We want to make sure we are ensuring parity. I was able to also reproduce on MacOS Desktop as an example:

<img width="333" alt="image" src="https://github.com/Expensify/App/assets/24947345/abdaf0af-770d-416d-9d97-f85d59dcc9e8">

<img width="342" alt="image" src="https://github.com/Expensify/App/assets/24947345/93603309-3ff3-4d3c-8c25-426652c10224">

**Comment by priya-zha** [comment]
I tested in on chrome (linux). Since it's related to tooltips, I could have tested it on Mac Os chrome or safari , but I don't have Mac to test them.

**Comment by shawnborton** [comment]
I think it makes sense to have the tooltip here out of consistency's sake. That being said, I think we just merged some code that touches this area so let's make sure it's still happening (cc @puneetlath)

**Comment by puneetlath** [comment]
I agree it makes sense to have the tooltip using the fallback details.

**Comment by mountiny** [comment]
@alitoshmatov please feel free to raise a PR, thanks!

**Comment by alitoshmatov** [comment]
@thesahindia Looks like icons are now empty and just have avatar for user
<img width="1423" alt="Screenshot 2023-07-10 at 10 41 26 PM" src="https://github.com/Expensify/App/assets/59907218/c9277932-385a-4b0e-89a8-7290a327330b">

The reason is that, `OptionsListUtils.getAvatarsForAccountIDs` is used, and only `accoundID`s are passed here, although we have login info.
https://github.com/Expensify/App/blob/e5b339455c4ccbaa4168e81562e0d86441f8db31/src/pages/workspace/WorkspaceInviteMessagePage.js#L197
And this method is setting default value as empty.

**Comment by alitoshmatov** [comment]
I think we should modify `OptionsListUtils.getAvatarsForAccountIDs` to also receive default values, or we can create a new method for this purpose

**Comment by alitoshmatov** [comment]
PR ready - https://github.com/Expensify/App/pull/22693

**Comment by NicMendonca** [comment]
@thesahindia will you be requesting payment via Expensify?

**Comment by NicMendonca** [comment]
@priya-zha can you please apply to the job? https://www.upwork.com/jobs/~01aa1190833be5a152

**Comment by NicMendonca** [comment]
@alitoshmatov you've been paid! 🎉

**Comment by NicMendonca** [comment]
bump @priya-zha @thesahindia ^^

**Comment by NicMendonca** [comment]
@priya-zha - reporting bonus $250
@alitoshmatov - Contributor - $1,500
@thesahindia - C+ $1,500 (paid via Expensify)

**Comment by NicMendonca** [comment]
@priya-zha just need you to accept the offer in Upwork please 🙏

**Comment by priya-zha** [comment]
@NicMendonca submitted the proposal. Apologies for the delay. I was out for a few days and didn't check the mail.

Thank you for the offer.

**Comment by NicMendonca** [comment]
@priya-zha you've been paid!

**Comment by NicMendonca** [comment]
Everyone has been paid!

@thesahindia don't forget to r[equest payment via Expensify. ](https://github.com/Expensify/App/issues/21946#issuecomment-1651624518)

> @priya-zha - reporting bonus $250
> @alitoshmatov - Contributor - $1,500
> @thesahindia - C+ $1,500 (paid via Expensify)

**Comment by JmillsExpensify** [comment]
Reviewed the details for @thesahindia. $1,500 approved for payment in NewDot based on the BZ summary above.
